tests/k8s/multi-node/tests/ipv4/ipv4.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+source "${dir}/../../helpers.bash"
+
```

Comment:
It's not unfinished, it's just a template so we can start using them if we want to create a dedicated IPv4 / IPv6 tests.